portfolio/src/main/webapp/contact.html

```diff
@@ -17,10 +17,23 @@
             </ul>
         </nav>
         
-    <div id="content">
+    <div id = "content">
+        <section class = "socials">
```

Comment:
The section doesn't need to have this blank space. You can do something like
 `<section class="socials"></section>` all on the same line